iocage/lib/Jail.py

```diff
@@ -384,6 +416,29 @@ def update_jail_state(self):
         except:
             self.jail_state = None
 
+    def _teardown_mounts(self):
+
+        mountpoints = list(map(
+            lambda mountpoint: f"{self.path}/root{mountpoint}",
+            [
+                "/dev/fd",
+                "/dev",
+                "/proc",
+                "/root/compat/linux/proc"
```

Comment:
This were only the ones potentially mounted during startup. All the other ones should be listed in the jail's fstab file that is read a few lines later.